.hlint.yaml

```diff
@@ -9,7 +9,26 @@
 # Specify additional command line arguments
 #
 # - arguments: [--color, --cpp-simple, -XQuasiQuotes]
-
+- arguments:
+  - '-XBangPatterns'
```

Comment:
I think this should be: `--ghc-opt -XBangPatterns --ghc-opt -XDeriveAnyClass ...`

Reply:
You probably mean a different file, but thanks, it helped me find the right thing. :+1:

The syntax I needed was ['--ghc-opt=-X...','--ghc-opt=-X',...]